be2-scala/README.md

```diff
@@ -0,0 +1,32 @@
+# student20_pop: be2-scala branch
+Proof-of-personhood, fall 2020: Scala language back-end
```

Comment:
student21_pop and spring 2021 🙂

Reply:
Done!

Same here. Many other changes will follow